app/(auth)/login/page.tsx

```diff
@@ -17,20 +16,21 @@ import { Button } from "@/components/ui/button";
 import { Input } from "@/components/ui/input";
 import { Label } from "@/components/ui/label";
 
+export const authMethods = ["google", "email", "linkedin", "passkey"] as const;
+
+export type AuthMethod = (typeof authMethods)[number];
+
```

Comment:
@mohdjami you cannot export this from a page.tsx in app router. 

instead just add them as normal `const` inthe page function 

here's the build error: 
```
Type error: Page "app/(auth)/login/page.tsx" does not match the required types of a Next.js Page.
  "authMethods" is not a valid Page export field.
```

Reply:
@mfts Fixed it.